px-context-browser.html

```diff
@@ -990,23 +1083,34 @@ <h1 class="epsilon caps weight--normal u-m0" on-click="toggleColumnBrowser">
       },
 
       /**
-      * Fires whenever the Context Browser is opened
-      * @event contextBrowserOpening
-      */
+       * Fires whenever the Context Browser is opened
+       * @event contextBrowserOpening
+       */
       /**
-      * Fires whenever the Context Browser is closed
-      * @event contextBrowserClosing
-      */
+       * Fires whenever the Context Browser is closed
+       * @event contextBrowserClosing
+       */
       /**
-      * toggle visbility of the column browser
-      * @method toggleColumnBrowser
-      */
-       toggleColumnBrowser: function() {
+       * toggle visbility of the column browser
+       * @method toggleColumnBrowser
+       */
+      toggleColumnBrowser: function(evt) {
         if (this.disabled) {
           this.showColumnBrowser = false;
         } else {
           this.showColumnBrowser = !this.showColumnBrowser;
         }
+        if (evt && evt.type === 'click') {
+          this.multiSelectAllowed = false;
+        }
+        // clear checkboxes when opening
+        if (this.showColumnBrowser) {
```

Comment:
For the comments related to 'toggleColumnBrowser', I can explain it to you in person. Will you be in office tomorrow?